pkg/systemd/terminal.scss

```diff
@@ -5,7 +5,7 @@
 .console-ct-container {
   block-size: 100%;
   display: grid;
-  grid-template-rows: auto 1fr;
+  grid-template-rows: auto auto 1fr;
```

Comment:
I'm still interested in what exactly that does. Does it need to change when moving the alert group up?

Reply:
Read the docs and from what I read I think i understand this: it's because we're adding another row. Before, the page only had two rows, so the first (header) was set to auto, and the 2nd (terminal) was set to fill up the rest of the page. I think adding in the extra auto makes it so that the alert takes the space it needs when it shows up, and it still makes sure that the terminal div is the one taking up the rest of the page.

Reply:
Yes, but what I don't understand is: In most cases we don't render an alert at all. So wouldn't the real terminal component then move from the "1fr" to the "auto" row? (and the third row not being used)

Reply:
Aah, of course! *slaps forehead* -- that goes hand in hand with the `<div className="ct-terminal-dir-alert">` which I was also wondering about. That's the "placeholder" for the second "auto".  All clear now!